templates/rust/common.tera

```diff
@@ -268,7 +361,7 @@ where
     A: Write,
 RegValueT<T>: Default,
 {
-    /// Init register with value returned by the closure.
+    /// Write register with value built from default register value
```

Comment:
I would change to 
`/// Write register with register value built from default register value`